src/components/util/logger.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import _ from 'underscore';
+import { isServerSide } from './misc';
+import * as Sentry from "@sentry/react";
+import { Integrations } from "@sentry/tracing";
+
+
```

Comment:
(Can create a global/module-level `let` variable or a `const` object/dictionary to preserve settings from initialization (i.e. `dsn`, `isInitialized`)).